scripts/create-bots-regex.js

```diff
@@ -1,12 +0,0 @@
-const fetch = require('node-fetch')
```

Comment:
Why removing the generation script here?

Reply:
Honestly I have no idea, maybe it wasn't needed to make the module work so I forgot about it, I've restored it.

Reply:
The idea was supposedly to have an easy way to generate the browser regexes so it isn't necessary to do it manually 😋